src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+	private Person personTagged;
+	private Tag tag;
+	private boolean isAdded;
+	public static ArrayList<Tagging> totalTaggings;
```

Comment:
Tagging class should not hold the Tagging list.  Tagging class is simply an object that associates a person to a tag.  So the proper design should be `AddressBook` holding the list of `Tagging`.